diagnostic.md

```diff
@@ -17,62 +17,70 @@ Which layer in the MVC pattern is used by the controller to fetch data?
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller tells the model what to do
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+I don't know what this asking, we do use use views in the MVC pattern, its the V.
```

Comment:
We don't use views on the back-end because we handle them on the front-end, its one thing that makes SPAs so cool.